src/main/java/banking/database/AccountQuery.java

```diff
@@ -0,0 +1,27 @@
+package banking.database;
+
+public enum AccountQuery {
```

Comment:
Why enum?

Reply:
Thought that's a good idea to separate queries from DAO class. 
Now I understand the problem of that decision, so moved all queries to AccountDAO as static final strings